src/jvmTest/kotlin/com/github/doyaaaaaken/kotlincsv/client/CsvFileWriterTest.kt

```diff
@@ -89,6 +90,15 @@ class CsvFileWriterTest : WordSpec() {
                 val actual = readTestFile()
                 actual shouldBe expected
             }
+            "write escaped field when a field contains quoteChar in it (quote all)" {
+                val rows = listOf(listOf("a", "\"b", "c"), listOf("d", "e", "f\""))
+                val expected = "\"a\",\"\"\"b\",\"c\"\r\n\"d\",\"e\",\"f\"\"\"\r\n"
+                csvWriter { quote { mode = WriteQuoteMode.ALL } }.open(testFileName) {
+                    writeRows(rows)
+                }
+                val actual = readTestFile()
+                actual shouldBe expected
+            }
             "write escaped field when a field contains delimiter in it" {
```

Comment:
I'd be happy to write additional test code written in below comment.
https://github.com/doyaaaaaken/kotlin-csv/issues/62#issuecomment-696770119

(Maybe `Customized CsvWriter` section is proper. 
https://github.com/doyaaaaaken/kotlin-csv/blob/ae8721271f805303118fe8acc90516d7565a2290/src/jvmTest/kotlin/com/github/doyaaaaaken/kotlincsv/client/CsvWriterTest.kt#L125 )

Reply:
Ah, missed that test section. Will change that 👍

Reply:
I added your test to the "Customized CsvWriter" section.